doc/source/user_guide/io.rst

```diff
@@ -1949,16 +1949,10 @@ Writing in ISO date format, with microseconds:
    json = dfd.to_json(date_format="iso", date_unit="us")
    json
 
-Epoch timestamps, in seconds:
-
-.. ipython:: python
-
-   json = dfd.to_json(date_format="epoch", date_unit="s")
-   json
-
 Writing to a file, with a date index and a date column:
 
 .. ipython:: python
+   :okwarning:
```

Comment:
I would rather we pass date_format="iso" in the doc examples so that warnings do not appear